lms/validation/authentication/_oauth.py

```diff
@@ -138,9 +138,4 @@ class OAuthTokenResponseSchema(RequestsResponseSchema):
 
     access_token = fields.Str(required=True)
     refresh_token = fields.Str()
-    expires_in = fields.Integer()
-
-    @marshmallow.validates("expires_in")
-    def validate_quantity(self, expires_in):  # pylint:disable=no-self-use
-        if expires_in <= 0:
-            raise marshmallow.ValidationError("expires_in must be greater than 0")
```

Comment:
This `@marshmallow.validates` method would need updating to raise `ExternalRequestError` instead of `ValidationError`, and it'd have to pass the `request`, `response` and `validation_errors` to `ExternalRequestError`. But I realised that there's no need for a `@marshmallow.validates` method here: you can just use Marshmallow's builtin `Range` validator in the `expires_in` field